client/src/components/common/Image/index.js

```diff
@@ -0,0 +1,41 @@
+import Image from 'material-ui-image';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CommonImage = ({
+  src,
+  width,
+  height,
+  animationDuration,
+  onClick,
+  alt,
+  style,
+}) => (
+  <Image
+    imageStyle={{ width, height }}
+    animationDuration={animationDuration}
+    onClick={onClick}
+    alt={alt}
+    style={style}
```

Comment:
There is ready props called image style to override image style , so I think its good to use it .
https://mui.wertarbyte.com/#image 
```suggestion
    imageStyle={style}
```

Reply:
there is two ready props "style and imageStyle",
* style is used to write any style you want.
* imageStyle to specify just width and height.

[Material UI Image](https://github.com/TeamWertarbyte/material-ui-image/blob/v3.1.0/README.md#material-ui-image-properties)

Reply:
ok, got it. thank you